scripts/api-check.sh

```diff
@@ -2,7 +2,12 @@
 
 exit_code=0
 
-./gradlew metalavaCheckCompatibilityDefaultsRelease || exit_code=$?
-./gradlew metalavaCheckCompatibilityCustomEntitlementComputationRelease || exit_code=$?
+# Run the api-dump.sh script to generate the API dump
+./scripts/api-dump.sh || exit_code=$?
+# Check if there are any dirty changes in git
+if ! git diff --quiet; then
+  echo "API dump has changes, please review and commit them."
```

Comment:
Would it be worth mentioning in this message how to generate the dump? something like the comment above
> "Run the api-dump.sh script to generate the API dump"

Perhaps it's too obvious but might be helpful for those of us not so familiar with it. FFTI